src/features/user/Account/PauseModal.tsx

```diff
@@ -78,25 +80,26 @@ export const PauseModal = ({
 
   const pauseDonation = async () => {
     setDisabled(true);
-    const bodyToSend = {
-      pauseType:
-        option == 'pauseForMonth' || option == 'pauseUntilDate'
-          ? 'custom-date'
-          : 'infinite', //custom-date | infinite
+    const pauseType: PauseType =
+      option === 'pauseForMonth' || option === 'pauseUntilDate'
+        ? 'custom-date'
+        : 'infinite';
```

Comment:
Why was this moved out? 

Perhaps we should follow the same approach for both payload properties?